preprocessors/openstreetmap/osm_service.py

```diff
@@ -660,17 +675,19 @@ def get_amenities(bbox_coord):
         amenities = server_config2(defaultServer, bbox_coord)
     except Exception:
         try:
-            error = 'Primary server not responding, so connecting server 1'
-            logging.error(error)
+            error = 'Pegasus Server not responding, so connecting VK Maps'\
```

Comment:
Please do not hard code the names of servers. This will just lead to a headache later on.

Reply:
All the necessary changes were effected.